### PR TITLE
Server version added to versionCmd 

### PR DESCRIPTION
**Pull request by anandrkskd**
Signed-off-by: anandrkskd <anandrkskd@gmail.com>
#603 
Previously only client version was displayed when version command was used.
The new update will show the client version as well as the server version.

### CODE REVIEW

File: mesheryctl/cmd/version.go [hunk 1]
```diff
@@ -20,8 +20,11 @@ import (
 	"github.com/spf13/cobra"
 )
 
-// Build - holds the build info
-var Build string
+// BuildClient - holds the build info of Client
+var BuildClient string
+
+// BuildServer - holds the build info of server
```
Comment (leecalcote) [inline mesheryctl/cmd/version.go]: Good commenting.

File: mesheryctl/main.go [hunk 2]
```diff
@@ -22,7 +22,8 @@ var version = "dev"
 var commitsha = "SHA"
 
 func main() {
-	cmd.Build = version
+	cmd.BuildClient = version
+	cmd.BuildServer = version
```
Comment (leecalcote) [inline mesheryctl/main.go]: It looks like you're setting the server version to always be the same of the client. The point of this feature is to acknowledge that in a given Meshery deployment, the two of these may differ in version number. `mesheryctl` needs to have a way of retrieving the server version from Meshery itself.

Comment (leecalcote) [reply mesheryctl/main.go]: @anandrkskd does this make sense?

Comment (anandrkskd) [reply mesheryctl/main.go]: Yes sir, Got it, But I had conversation with @girishranganathan about it on the development meeting, and he asked me to just add the server version same as client for now and when we will develop the api for meshery server then we will replace it with actual server version and for now we are also publishing meshery and mesheryctl binary together so both of them carry the same version.
But I guess, for now we can add a new serververion tag in goreleaser for server version in versionCmd.

**Comment by leecalcote** [comment]
Makes sense, @anandrkskd. Will you open up an issue to track the need to implement this server-side?

**Comment by leecalcote** [comment]
And maybe another to track the need to update goreleaser?

**Pull request merged**